src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DefaultTagHelperDescriptionFactoryTest_ClassifiedTextElement.cs

```diff
@@ -0,0 +1,273 @@
+            AssertExpectedClassification(runs[9], ".", RazorPredefinedClassificationTypeNames.Punctuation);
+            AssertExpectedClassification(runs[10], "SomeProperty", RazorPredefinedClassificationTypeNames.Identifier);
+            AssertExpectedClassification(runs[11], Environment.NewLine, RazorPredefinedClassificationTypeNames.WhiteSpace);
+            AssertExpectedClassification(runs[12], "Uses ", RazorPredefinedClassificationTypeNames.Text);
+            AssertExpectedClassification(runs[13], "List", RazorPredefinedClassificationTypeNames.Type);
+            AssertExpectedClassification(runs[14], "<", RazorPredefinedClassificationTypeNames.Punctuation);
+            AssertExpectedClassification(runs[15], "List", RazorPredefinedClassificationTypeNames.Type);
+            AssertExpectedClassification(runs[16], "<", RazorPredefinedClassificationTypeNames.Punctuation);
+            AssertExpectedClassification(runs[17], "string", RazorPredefinedClassificationTypeNames.Keyword);
+            AssertExpectedClassification(runs[18], ">", RazorPredefinedClassificationTypeNames.Punctuation);
+            AssertExpectedClassification(runs[19], ">", RazorPredefinedClassificationTypeNames.Punctuation);
+            AssertExpectedClassification(runs[20], "s", RazorPredefinedClassificationTypeNames.Text);
+        }
+
+        [Fact]
+        public void TryCreateTooltip_ClassifiedTextElement_Attribute_MultipleAssociatedAttributes_ReturnsTrue()
+        {
+            // Arrange
+            var descriptionFactory = new DefaultTagHelperTooltipFactory(GetLanguageServer(supportsVisualStudioExtensions: true));
+            var associatedAttributeDescriptions = new[]
+            {
+                new BoundAttributeDescriptionInfo(
+                    returnTypeName: "System.String",
+                    typeName: "Microsoft.AspNetCore.SomeTagHelpers.SomeTypeName",
+                    propertyName: "SomeProperty",
+                    documentation: "<summary>Uses <see cref=\"T:System.Collections.List{System.String}\" />s</summary>"),
+                new BoundAttributeDescriptionInfo(
+                    propertyName: "AnotherProperty",
+                    typeName: "Microsoft.AspNetCore.SomeTagHelpers.AnotherTypeName",
+                    returnTypeName: "System.Boolean?",
+                    documentation: "<summary>\nUses <see cref=\"T:System.Collections.List{System.String}\" />s\n</summary>"),
+            };
+            var attributeDescription = new AggregateBoundAttributeDescription(associatedAttributeDescriptions);
+
+            // Act
+            var result = descriptionFactory.TryCreateTooltip(attributeDescription, out RazorClassifiedTextElement classifiedTextElement);
+
+            // Assert
+            Assert.True(result);
+
+            var runs = classifiedTextElement.Runs.ToArray();
+            Assert.Equal(39, runs.Length);
```

Comment:
The `Assert.Collection` suggestion above should avoid you needing to do this or the ToArray above